packages/core/contracts/optimistic-assertor/implementation/OptimisticAssertor.sol

```diff
@@ -268,6 +269,12 @@ contract OptimisticAssertor is Lockable, OptimisticAssertorInterface, Ownable {
         return SovereignSecurityManagerInterface(ssm).getAssertionPolicies(assertionId);
     }
 
+    function _isDisputeAllowed(bytes32 assertionId) internal view returns (bool) {
+        address ssm = assertions[assertionId].sovereignSecurityManager;
+        if (ssm == address(0)) return true;
+        return SovereignSecurityManagerInterface(ssm).isDisputeAllowed(assertionId, msg.sender);
```

Comment:
Note that this adds gas costs for all SSM implementations even if not using disputer whitelist. Alternative would be to store additional SSM policy on whether to call `isDisputeAllowed` at assertion time.

Reply:
we could also add add a prop at assertion time to the policy whether the OA should check with the asserter on `isDisputeAllowed`. so it's off by default unless you explicitly want to check this.

Reply:
wdyt? this could be in `ssmSettings`?

Reply:
Added `validateDisputers` to SSM policies that is checked before calling `isDisputeAllowed`. By default this `validateDisputers` is disabled and used only in this SSM implementation